vispy/visuals/volume.py

```diff
@@ -382,11 +430,11 @@
         // Refine search for max value, but only if anything was found
         if ( maxi > -1 ) {
             // Calculate starting location of ray for sampling
+            vec3 loc = start_loc + step * maxi;
```

Comment:
can you elaborate on this change?

Reply:
Weird, my response to this ended up in https://github.com/vispy/vispy/pull/2550/files/f858c92ab13fb572b2ec070e436c08ea5b1f0944#r1681378506